scripts/convert-qpp-to-measures.js

```diff
@@ -42,6 +36,30 @@ var CEHRT_ELIGABLE_IA_MEASURE_IDS = [
   'IA_PM_4',
   'IA_PSPA_16'
 ];
+// Utils
+var isInverseBenchmarkRecord = require('./../util/is-inverse-benchmark-record');
+// Data
+var benchmarksData   = fs.readFileSync(path.join(__dirname, './../data/historical-benchmarks/2015.csv'), 'utf8');
+var benchmarkRecords = parse(benchmarksData, {columns: BENCHMARK_CSV_COLUMNS, from: 4});
+// Commandline arguments
+var category = process.argv[2] || 'ia';
+// Variables
+var qualityIdToIsInverseMap = {};
+var qpp = '';
+
+benchmarkRecords.forEach(function(record) {
+  // NOTE: There is a special case, e.g. qualityId 378 EHR, which has decile
```

Comment:
Oh geez. So quality IDs are not unique? I'd explain this a little more because right now it seems dependent on order - explain that all benchmarks for a particular measure will have the same isInverse flag, some benchmark records will have no data so the flag will be false and in these cases when we come up with a benchmark with the same quality id, we always want to double check.